src/frontend/js/components/CourseProductCertificateItem/index.spec.tsx

```diff
@@ -0,0 +1,140 @@
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
+import faker from 'faker';
+import fetchMock from 'fetch-mock';
+import { PropsWithChildren } from 'react';
+import { IntlProvider } from 'react-intl';
+import {
+  CertificationDefinitionFactory,
+  ContextFactory as mockContextFactory,
+  OrderLiteFactory,
+} from 'utils/test/factories';
+import JoanieApiProvider from 'data/JoanieApiProvider';
+import { CertificateDefinition, OrderLite } from 'types/Joanie';
+import { handle } from 'utils/errors/handle';
+import CertificateItem from '.';
```

Comment:
my usual comment, it's nicer when import are ordered in alphabetical order, 
ditto on all files

Reply:
that explains then